client/src/related/productCard.jsx

```diff
@@ -21,16 +13,28 @@ class ProductCard extends React.Component {
       iconSrc = 'https://i.dlpng.com/static/png/6978437_preview.png';
     }
     return (
-      <div className="related_productCard" onClick={(event) => {
-        event.preventDefault();
-        this.props.handleCardClick(this.props.productInfo.id);
-      }}>
-        <img className='related_productCardImage' src={this.props.productInfo.image || 'https://static.thenounproject.com/png/1103191-200.png'}></img>
-        <img className='related_icon' src={iconSrc}></img>
-        <p className='related_details'>{this.props.productInfo.category}</p>
-        <p className='related_details'>{this.props.productInfo.name}</p>
-        <SalePrice original_price={this.props.productInfo.original_price} sale_price={this.props.productInfo.sale_price} />
-        <StarRating rating={this.props.productInfo.avgReview} />
+      <div className="related_productCard">
+        <img className='related_productCardImage' src={this.props.productInfo.image || 'https://static.thenounproject.com/png/1103191-200.png'} onClick={(event) => {
```

Comment:
Do you need to pass in the event on 17,21,29?  Is it just to use preventDefault?

Reply:
That's why I was passing it in, but now that I'm reading it looks like I got a little event.preventDefault happy? I've had some bad experiences forgetting it so I've just been including it in all my click handlers... but I don't think it's actually necessary unless trying to prevent default HTML behavior (forms, buttons), yes?

Reply:
I think so, yes.  I think React also treats events a little bit differently ( https://reactjs.org/docs/events.html ) but I am not sure if that applies to how you were using it here.  I have also had issues with it.